docs/intro/go.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 2
+description: Install the Go programming language
+---
+
+# Install Go
+
+In this tutorial, you will install the Go programming language (Golang) on your local computer. Follow the instructions for your operating system:
+
+## macOS
+
+* Download the latest MacOS installer package from <https://golang.org/dl/>.
```

Comment:
```suggestion
* Download the latest macOS installer package from [Go downloads](https://golang.org/dl/).
```

Reply:
I was curious, so I verified "macOS" from https://support.apple.com/en-us/HT201260 and a few other Mac doc peeks. 
I know that URLs and URLs in angle brackets will automatically get turned into links. However, our style is to use link text, so let's follow the Google developer documentation style guidance to use [descriptive link text](https://developers.google.com/style/link-text).